mantidproject/mantid
# Support for loading SHARP data

This PR enables SHARP data to be loaded using already available solution in `LoadILLTOF2` algorithm.

**To test:**



1. Using SHARP data added to the UnitTest (ILL/SHARP):
2. Load 000102.nxs (non-TOF) and 000103.nxs (TOF) data
3. Verify that sample logs are set, that instrument viewer connects spectra to detectors, and no strange patterns are visible. 

Part of #30902 .

Release notes added.



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
